clients/fides-js/src/lib/cookie.ts

```diff
@@ -52,7 +56,8 @@ export interface FidesCookie {
   consent: CookieKeyConsent;
   identity: CookieIdentity;
   fides_meta: CookieMeta;
-  tcString?: string;
+  tc_string?: string;
```

Comment:
I realized we had a mix of camel and snake case in this cookie... I went with snake case since that makes things a little easier to work with on the tc string side. `createdAt` and `updatedAt` in `fides_meta` are still camel case though, but I think we'd have to do a migration to change those and that doesn't sound fun! I figured I'd change `tcString` to snake case now, before anyone is using this feature yet